Bottom: 679a81a082ef84f6d6c397841744d42245b0a328
Top:    42c7973398e602e65e87b8abf5450c440eaf08f4
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-07-27 17:20:43 +0800

Refresh of tun-xdp-support

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index b7e6bd6..13abe23 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1031,7 +1031,6 @@ static int tun_xdp_set(struct net_device *dev, struct bpf_prog *prog,
 		bpf_prog_put(old_prog);
 	rcu_assign_pointer(tun->xdp_prog, prog);
 
-
 	if (prog) {
 		prog = bpf_prog_add(prog, 1);
 		if (IS_ERR(prog))
